### PR TITLE
Iteration 5 methods

### PR DESCRIPTION
**Pull request by msimon42**

### CODE REVIEW

File: lib/season_module.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require 'pry'
```
Comment (hale4029) [inline lib/season_module.rb]: remove this maybe so it doesn't mess with rspec

File: lib/game_team.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@ def self.create(file_path)
                                     row[:settled_in],
                                     row[:head_coach],
                                     row[:goals].to_i,
-                                    row[:shots],
+                                    row[:shots].to_i,
```
Comment (hale4029) [inline lib/game_team.rb:36]: we might need to adjust game_team test for this change. did you run rake with these changes?

File: test/season_test.rb [hunk 3]
```diff
@@ -13,31 +13,31 @@ def setup
   end
 
   def test_biggest_bust
-    assert_equal "Atlanta United", @stat_tracker.biggest_bust('20132014')
+    assert_equal 0, @stat_tracker.biggest_bust('20122013')
```
Comment (hale4029) [inline test/season_test.rb]: i think you forgot to pull/merge to master and it is trying to change these back to a previous version

File: test/season_test.rb [hunk 4]
```diff
@@ -13,31 +13,31 @@ def setup
...
   end
 
-  def test_biggest_surprise
-    assert_equal "Seattle Sounders FC", @stat_tracker.biggest_surprise('20132014')
+  def test_biggest_bust
```
Comment (hale4029) [inline test/season_test.rb]: same.

File: test/season_test.rb [hunk 5]
```diff
@@ -13,31 +13,31 @@ def setup
...
-  def test_most_accurate_team
+  def test_find_coach
+    assert_equal 'Peter DeBoer', @stat_tracker.find_coach('1', '20122013')
   end
 
-  def test_least_accurate_team
-  end
-
-  def test_most_tackles
-    assert_equal 'Houston Dynamo', @stat_tracker.most_tackles('20132014')
```
Comment (hale4029) [inline test/season_test.rb]: diddo

File: test/season_test.rb [hunk 6]
```diff
@@ -13,31 +13,31 @@ def setup
...
+  def test_most_accurate_team
+    assert_equal 'Atlanta United', @stat_tracker.most_accurate_team('20122013')
   end
 
-  def test_fewest_tackles
```
Comment (hale4029) [inline test/season_test.rb]: diddo

**Review by hale4029 — CHANGES_REQUESTED** [review]
Looks good! Please see comments before merge.

File: lib/game_team.rb (hunk 2 above)
Comment (msimon42) [reply lib/game_team.rb:36]: It looks like we aren't failing any of those tests.

**Review by hale4029 — APPROVED** [review]
perfect, thanks for reviewing.

**Pull request merged**